aws/cloudformation/iam.yml.erb

```diff
@@ -336,6 +338,13 @@ roles = {
     ManagedPolicyArns: [
       'arn:aws:iam::aws:policy/AWSBillingReadOnlyAccess',
     ]
+  },
+  ProductFullAccessRole: {
+    RoleName: 'Product_FullAccess',
```

Comment:
Could we name this `ProductManager_FullAccess`? I think someone who saw this Role in the console or referenced in code might think it’s for our application product and not for a type of employee role.

Reply:
Yes, that makes sense and keeps the language the same `Engineer/Contributor/ProductManager`

The 'billing' role violates this, but we can align that too.